src/test/java/com/opencart/tests/userLogin/UserLoginTest.java

```diff
@@ -0,0 +1,67 @@
+package com.opencart.tests.userLogin;
+
+import com.opencart.navigation.Navigation;
+import com.opencart.steps.LoginPageBL;
+import com.opencart.steps.MainPageBL;
+import com.opencart.tests.BaseTest;
+import org.testng.annotations.Test;
+
+import static com.opencart.enums.URLs.BASE_URL;
+
+public class UserLoginTest extends BaseTest {
+
+    @Test
+    public void userLoginTestWithValidInput() {
+        new Navigation().navigateToURrl(BASE_URL.getValue());
+        MainPageBL mainPageBL = new MainPageBL();
+        LoginPageBL loginPageBL = mainPageBL.getHeaderPageBL()
+                .clickOnMyAccountButton()
+                .clickOnLoginButton()
+                .login();
+        loginPageBL.verifyLogin();
+    }
+
+    @Test
+    public void userLoginWithoutPassword() {
+        new Navigation().navigateToURrl(BASE_URL.getValue());
+        MainPageBL mainPageBL = new MainPageBL();
+        LoginPageBL loginPageBL = mainPageBL.getHeaderPageBL()
+                .clickOnMyAccountButton()
+                .clickOnLoginButton()
+                .loginWithoutPassword();
```

Comment:
Please, perform the same improvements for Login flow (Move corresponding models to login repository and use one .login() method putting the corresponding LoginModel as a parameter for it)